common/flatpak-installation.c

```diff
@@ -1093,6 +1177,20 @@ flatpak_installation_list_installed_refs_for_update (FlatpakInstallation *self,
       if (remote_commit != NULL &&
           g_strcmp0 (remote_commit, local_commit) != 0)
         g_ptr_array_add (updates, g_object_ref (installed_ref));
+      else if (flatpak_ref_get_kind (FLATPAK_REF (installed_ref)) == FLATPAK_REF_KIND_APP)
```

Comment:
Runtimes can also have similar extensions, do we not need to do a similar thing there? For instance, mark the runtime as needs update when there is a new nvidia driver extension for it needed?

Reply:
Re work-ed the code so that runtimes related refs are not ignored. Honestly, I didn't know runtimes can have extension refs too.